python/basix/ufl_wrapper.py

```diff
@@ -1229,6 +1230,8 @@ def create_element(family: _typing.Union[_basix.ElementFamily, str], cell: _typi
         gdim: The geometric dimension of the cell.
 
     """
+    if isinstance(cell, _ufl.cell.Cell):
```

Comment:
Could try/except be used here? Nice to avoid C++ style type checks if we can.

Reply:
I can't see a way to do it that isn't a complete mess.

Reply:
What if we only accept a `basix.CellType`? A lot of merit to there being only one way to do something.

Reply:
`create_element` is a convenience function in the UFL wrapper to make it more user friendly to create elements. I've aimed to make the interface of this function as similar to the current UFL `FiniteElement` constructor as possible and it saves most users from having to write garbage like: 

```python
element = basix.ufl_wrapper.Basix_element(basix.create_element(basix.ElementFamily.P, basix.CellType.triangle, 2, basix.LagrangeVariant.equispaced))
```

Reply:
But potentially accepting strings is enough and we don't need this additional option for input